alpha/alpha.ino

```diff
@@ -8,95 +8,168 @@
 
 bool powerSaveTest = false;
 
+bool SavePower = false;
+int stackSize = 1000;
```

Comment:
is there a default stack size somewhere?

Reply:
I just set an arbitrarily high value then use uxTaskGetStackHighWaterMark to fine the required stack size